cmd/calyptia/k8s_client.go

```diff
@@ -76,6 +75,22 @@ func (client *k8sClient) createOwnNamespace(ctx context.Context) (*apiv1.Namespa
 	}, metav1.CreateOptions{})
 }
 
+func (client *k8sClient) createSecret(ctx context.Context, name string, value []byte) (*apiv1.Secret, error) {
+	return client.CoreV1().Secrets(client.namespace).Create(ctx,
+		&apiv1.Secret{
+			TypeMeta: metav1.TypeMeta{
+				Kind: "Secret",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
```

Comment:
Is missing the labels.
```suggestion
				Name:   name,
				Labels: client.labelsFunc(),
```